code/modules/assembly/flash.dm

```diff
@@ -114,40 +114,63 @@
 /obj/item/assembly/flash/proc/flash_end()
 	set_light_on(FALSE)
 
-
+///Awfully bad proc that flashes carbons. this really fucking needs a refactor
```

Comment:
This comment does not meet our [documentation standards](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-proc), please update this.